author/concepts/metanorma_workflow.adoc

```diff
@@ -0,0 +1,89 @@
+== The Metanorma workflow
+
+The Metanorma workflow consists of six phases:
+
+. Document creation
+. Editing the document
+. Validation
+. Output generation
+. Review
+. Publication
+
+
+.The Metanorma Workflow from start to finish 
+image::../assets/author/concepts/Metanorma_Workflow.png[The Metanorma workflow]
+
+== Document creation
+
+You can create a new standard document either by:
+
+* Creating a new, empty AsciiDoc file
+* Using a template
+* Importing a Word Document
+
+*New file*
+
+The easiest way is to create a document from scratch, by creating an empty AsciiDoc file (`.adoc`) in the desired folder. 
+
+*Templates*
+
+Templates get you started faster, as they provide the structure of a standard document already. If your organization provides templates, you can create a new document based on a template using the command line. 
+
```

Comment:
Point to where this is documented.

Reply:
There's a related links section so the reader isn't disturbed in the reading flow.